client/hooks/useJob.ts

```diff
@@ -109,10 +116,7 @@ const useJob = ({ category, children }) => {
         await updateStatus();
     };
 
-    const updateStatus = async (e) => {
-        if (typeof e !== "undefined") {
-            e.stopPropagation();
-        }
+    const updateStatus = async () => {
```

Comment:
Any idea why this event code was here to begin with? Just want to be sure I understand why it's being removed.

Reply:
I'm going to reviert the change.